terraform/workspace-variables/review.tfvars

```diff
@@ -25,7 +25,7 @@ documents_s3_bucket_force_destroy = true
 
 # Gov.UK PaaS
 paas_space_name                     = "teaching-vacancies-review"
-paas_postgres_service_plan          = "tiny-unencrypted-11"
+paas_postgres_service_plan          = "small-11"
```

Comment:
Make sure to revert before merging

Reply:
This one might be a bit tricky...? I will revert in a separate PR.